js/plugins/vertexai/src/modelgarden/openai_compatibility.ts

```diff
@@ -42,14 +42,84 @@ import {
   type CompletionChoice,
 } from 'openai/resources/index.mjs';
 
+/**
+ * See https://platform.openai.com/docs/api-reference/chat/create.
+ *
+ * TODO: topK is not supported and some of the other common config options
+ * have different names in the above doc. Eg: max_completion_tokens.
+ * Update to use the parameters in above doc.
```

Comment:
Suggest moving the todo out of the tsdoc, so its not picked up my the api refs.

Reply:
Done.